### ISSUE TITLE
Icon request:  fa-internal-link

### ISSUE THREAD

**Issue by nathG** [FortAwesome/Font-Awesome#3972]
Hello,

We would love the opposite of  fa-external-link: same icon with arrow from outside the box to inside the box!

Thanks a lot in advance!

**Comment by jeremyswright** [comment]
Yes, that would be pretty useful.  Any way I can help produce it?

**Comment by rogertheshrubber** [comment]
+1

**Comment by ryankane** [comment]
Here is a variant of the requested change to `fa-external-link` in Issue #10486 

![internal-link](https://cloud.githubusercontent.com/assets/1094151/21902459/fb6e725c-d8c9-11e6-8547-0cb5891f4d95.png) ![external-link](https://cloud.githubusercontent.com/assets/1094151/21901794/288a2522-d8c7-11e6-8ed8-c895794d20ae.png)

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 15 15">
  <g id="fa-internal-link">
    <g fill="none" stroke="#000" stroke-width="1px">
      <path id="fa-internal-link-box-border" d="m 8 3.5 h -4.5 v 8 h 8 l 0 -4.5" />
      <g id="fa-internal-link-arrow">
        <path id="fa-internal-link-arrow-head" d="m 5.5 6 v 3.5 h 3.5" />
        <path id="fa-internal-link-arrow-tail" d="m 10 5 -4.5 4.5" />
      </g>
    </g>
  </g>
</svg>
```

**Comment by nhansum** [comment]
Would love to see this one one too. +1

**Comment by leronamin** [comment]
Me too. +1

**Comment by cstephe** [comment]
I'd love to see this as well,  its very helpful in a table/grid situation where the user may not know they can navigate using the table.

here are a few varieties of existing v5 icons that kinda do this as well, but are repurposed so you basically highjack them.

<img width="897" alt="Screen Shot 2020-02-21 at 10 05 50 AM" src="https://user-images.githubusercontent.com/311854/75063011-c67c0480-5498-11ea-86d4-81ba923988c3.png">

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Hey there!

We've made the decision to move our icon requests flow to **GitHub Discussions**, as opposed to issue tracking. No more +1s or thumbs-ups needed. There's an upvote system built in.

As part of this transition, we are closing existing icon requests with lower comment or reaction counts, but **we are not saying no** to your request. (We know some of these have been open since before I had gray hair. And I've had gray hair for a LONG time.)

**Could you do us a favor? If this icon request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
→ https://github.com/FortAwesome/Font-Awesome/discussions/new?category=icon-requests

And listen. We know this is annoying. We would batch convert them to discussions if we could, but Github's removed that ability earlier this year. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worries at all. I love doing things twice.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. This request already HAS an auto-response saying that you're refining the workflow. What do you mean you're refining your refinement? That's a bunch of bull-phooey.
* No biggie. Thanks for letting us know what's up.
